scripts/validate_docstrings.py

```diff
@@ -531,6 +537,11 @@ def validate_one(func_name):
         examples_errs = doc.examples_errors
         if examples_errs:
             errs.append('Examples do not pass tests')
+        examples_codes = doc.examples_codes
+        if 'import numpy' in examples_codes:
+            errs.append('Examples should not have `import numpy` ')
+        if 'import pandas' in examples_codes:
+            errs.append('Examples should not have `import pandas` ')
```

Comment:
I think the error could be more descriptive. `pandas does not need to be imported in the examples, as it's assumed to be already imported as pd`. I think something like that provides more useful information to devs facing this error.